Client/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -1019,22 +1037,41 @@ std::variant<bool, CLuaMultiReturn<float, float, float>> CLuaPedDefs::GetElement
     return false;
 }
 
-std::variant<bool, CLuaMultiReturn<float, float, float>> CLuaPedDefs::GetElementBoneRotation(lua_State* const luaVM, CClientPed* entity, std::int32_t boneId)
+std::variant<bool, CLuaMultiReturn<float, float, float>> CLuaPedDefs::GetElementBoneRotation(lua_State* const luaVM, CClientPed* entity, std::uint32_t boneId)
 {
+    if (boneId > BONE_RIGHTFOOT)
+    {
+        throw LuaFunctionError("Invalid bone ID");
+    }
+
     float    yaw = 0.0f, pitch = 0.0f, roll = 0.0f;
     CEntity* theEntity = entity->GetGameEntity();
     if (theEntity && theEntity->GetBoneRotation(static_cast<eBone>(boneId), yaw, pitch, roll))
         return std::make_tuple(yaw, pitch, roll);
     return false;
 }
 
-bool CLuaPedDefs::SetElementBoneMatrix(lua_State* const luaVM, CClientPed* entity, std::int32_t boneId, CMatrix boneMatrix)
+std::variant<bool, CLuaMultiReturn<float, float, float, float>> CLuaPedDefs::GetElementBoneQuaternion(lua_State* const luaVM, CClientPed* entity, std::uint32_t boneId)
+{
+    if (boneId > BONE_RIGHTFOOT)
+    {
+        throw LuaFunctionError("Invalid bone ID");
+    }
```

Comment:
You can remove the braces here

Reply:
I'm sorry, I won't be home for 3 weeks so I'm unable to change that

Reply:
Probably, during this time, the PR will be waiting for you, so no worries. You can handle it when you return